### PR TITLE
Persisted flags for release 1.31.0

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

Sets the flags that will be included in the 1.31.0 release.

## How did you test this code?
TBD

### CODE REVIEW

File: posthog/settings.py [hunk 1]
```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
     # Add hard-coded feature flags for static self-hosted releases here
     "5440-multivariate-support",
     "new-paths-ui-edge-weights",
+    "sigma-analysis",
```
Comment (paolodamico) [inline posthog/settings.py:122]: CC @neilkakkar

File: posthog/settings.py [hunk 2]
```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
...
+    "new-sessions-player",
```
Comment (paolodamico) [inline posthog/settings.py:123]: Ok to release @rcmarron ?

File: posthog/settings.py [hunk 3]
```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
...
+    "new-sessions-player-events-list",
+    "938-breakdown-by-multiple-properties",
```
Comment (paolodamico) [inline posthog/settings.py:125]: @pauldambra do we want to include in the release yet?

**Comment by paolodamico** [comment]
Outstanding flags to figure out:
- `5733-funnel-vertical-breakdown` @alexkim205 
- `new-paths-ui-edge-weights` @EDsCODE  / @neilkakkar

File: posthog/settings.py [hunk 4]
```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
...
+    "session-insight-removal",
+    "funnel-simple-mode",
```
Comment (paolodamico) [inline posthog/settings.py:127]: For future context. This experiment is still ongoing on Cloud, but results are promising. Let's get self-hosted users on this to see how performance changes.

File: posthog/settings.py (hunk 1 above)
Comment (neilkakkar) [reply posthog/settings.py:122]: This is released to everyone on Cloud, so all good! I'd actually want to just get rid of the FF.

File: posthog/settings.py (hunk 3 above)
Comment (pauldambra) [reply posthog/settings.py:125]: Ah, not yet... Need to roll it out to beta users on cloud still :/

**Comment by alexkim205** [comment]
@paolodamico check out https://github.com/PostHog/posthog/pull/7654#pullrequestreview-829576494. Once thats merged we can release vertical breakdowns.

File: posthog/settings.py (hunk 2 above)
Comment (rcmarron) [reply posthog/settings.py:123]: Yeah, this feature flag has been removed and is live for everyone.

**Comment by paolodamico** [comment]
Not sure what's going on with those Docker builds, but everything looks good, merging.

**Pull request merged**